### ISSUE TITLE
Same Error as issue  #1745: install previous version v2.3.4 not possible

### ISSUE THREAD

**Issue by annecar** [satijalab/seurat#1768]
Hi 
I have the same problem as #1745 installing the previous version v2.3.4, running the command: 
install.packages('Seurat', repo = c('satijalab.org/ran/', getOption("repos")))

installs Seurat 3.0;  i re-installed R3.6 but this did not help. 

and using the command:

devtools::install_version(package = 'Seurat', version = package_version('2.3.4'))

fails, because of the following error: 
Error: Failed to install 'unknown package' from URL:
  (converted from warning) installation of package ‘hdf5r’ had non-zero exit status

Do you have another idea how o do it? Or what could be wrong? 
Thanks

**Comment by andrewwbutler** [comment]
Hi,

We've posted another solution [here](https://satijalab.org/seurat/install.html#previous) that should get around many of the errors people have been reporting. Give it a try and see if it resolve the issue.

**Comment by annecar** [comment]
Thanks a lot! Helped! 




Anne Eugster, PhD
Head Single Cell Unit/
Scientist AG Bonifacio

Technische Universität Dresden
DFG-Center for Regenerative Therapies Dresden
Cluster of Excellence
Fetscherstr. 105
01307 Dresden

Tel: +49 (0)351-458-821 41
Fax: +49 (0)351-458-821 09
Email: a.eugster@crt-dresden.de 
Webpage: www.crt-dresden.de

**Comment by MichaelStber** [comment]
Dear @andrewwbutler ,
dear @annecar ,

I had the same problem but when I tried to run the command from Seurat website to install Seurat v 2.3.4 (with R version 3.6.0) I got the following error after the packages were unpacked

> source("http://bit.ly/archived-seurat")

Installing package into ‘C:/R-libs1’
(as ‘lib’ is unspecified)
Error in install.packages : cannot install macOS binary packages on Windows

Does anyone has an idea what I could do in this case?

**Comment by mojaveazure** [comment]
Hi Michael,

That's a bug in our script. We've updated the script and installation instructions. Please run

```R
source("https://z.umn.edu/archived-seurat")
```

**Comment by bschilder** [comment]
Ran into the following issue when I ran the script:

```
> source("https://z.umn.edu/archived-seurat")

Downgrading Seurat from v4.0.0 to v2.3.4Installing packages into ‘/home/bschilder/R/x86_64-pc-linux-gnu-library/4.0’
(as ‘lib’ is unspecified)
Warning in install.packages :
  package ‘SDMTools’ is not available (for R version 4.0.2)
Warning in install.packages :
  dependency ‘multtest’ is not available
also installing the dependency ‘mutoss’

trying URL 'https://cloud.r-project.org/src/contrib/mutoss_0.1-12.tar.gz'
Content type 'application/x-gzip' length 991412 bytes (968 KB)
==================================================
downloaded 968 KB

trying URL 'https://cloud.r-project.org/src/contrib/metap_1.4.tar.gz'
Content type 'application/x-gzip' length 917683 bytes (896 KB)
==================================================
downloaded 896 KB

ERROR: dependency ‘multtest’ is not available for package ‘mutoss’
* removing ‘/home/bschilder/R/x86_64-pc-linux-gnu-library/4.0/mutoss’
Warning in install.packages :
  installation of package ‘mutoss’ had non-zero exit status
ERROR: dependency ‘mutoss’ is not available for package ‘metap’
* removing ‘/home/bschilder/R/x86_64-pc-linux-gnu-library/4.0/metap’
Warning in install.packages :
  installation of package ‘metap’ had non-zero exit status

The downloaded source packages are in
	‘/tmp/Rtmpxjy0kp/downloaded_packages’
Error in install.packages : Updating loaded packages
Attaching SeuratObject

Restarting R session...
```

Seurat is still v4.0.0 after R restart:

![Screenshot 2021-04-15 at 15 01 24](https://user-images.githubusercontent.com/34280215/114882065-79c15e80-9dfb-11eb-96b9-6433ae91be9a.png)

**Comment by bschilder** [comment]
Managed to install the deps by manually running `BiocManager::install("multtest")` first, but Seurat version remains 4.0.0
